cmd/sbctl/reset.go

```diff
@@ -150,6 +207,7 @@ func RunReset(cmd *cobra.Command, args []string) error {
 func resetKeysCmdFlags(cmd *cobra.Command) {
 	f := cmd.Flags()
 	f.VarPF(&resetCmdOpts.Partial, "partial", "p", "reset a partial set of keys")
+	f.StringVarP(&resetCmdOpts.CertFiles, "cert-files", "c", "", "optional paths to certificate file to remove from db (seperate individual paths by ';')")
```

Comment:
This may not only be db, but also other variables, right? Even though the function is called the same, this seems a bit confusing to me from the wording

Reply:
right, db is referring to the cert database of any hierarchy,  the wording is a bit off there.